renovate.json

```diff
@@ -1,121 +1,39 @@
 {
-  "enabled": true,
   "extends": [
-    ":dependencyDashboard",
+    "github>defenseunicorns/uds-common//config/renovate.json5",
     ":semanticCommits",
     ":semanticCommitTypeAll(chore)"
   ],
-  "forkProcessing": "enabled",
-  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
-  "branchConcurrentLimit": 0,
-  "prConcurrentLimit": 0,
-  "prHourlyLimit": 0,
```

Comment:
I think we will want to keep these 3 still - by default renovate has some rate limiting and with the amount of dependencies we have on core it might limit our ability to see PRs. Common config doesn't have any overrides to the limits.